baremetal/metal3datatemplate_manager.go

```diff
@@ -179,8 +179,9 @@ func (m *DataTemplateManager) UpdateDatas(ctx context.Context) (bool, bool, erro
 				continue
 			}
 		}
-
+		m.Log.Info("Initiate updating data of claim", "Metal3DataClaim", dataClaim.Name, "Metal3DataTemplate", m.DataTemplate.Name)
```

Comment:
Should we maybe use some other log level for this? I am not sure how spammy it would be, but it does feel closer to "debug" than "info" for me.